doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -861,7 +861,7 @@ are applied using the tactic :n:`rewrite_strat @rewstrategy`.
 :n:`@rewstrategy ; @rewstrategy`
    composition
 
-:n:`choice {+ @rewstrategy }`
+:n:`choice {+ @rewstrategy_atom }`
```

Comment:
This is now
```suggestion
:n:`choice {+ @rewstrategy0 }`
```
right?

Reply:
Indeed, this is incorrect, but this shouldn't be fixed by editing the rst.  I think Hugo didn't include the rst file changes in his last commit.  If you look in `orderedGrammar`, it has rewstrategy0 in this production.  Running `make doc_gram_rsts` will update the rst.  Then make sure it's included in the commit.

Reply:
This one is not generated and had to be done manually. Also fixed.